interactive.py

```diff
@@ -7,7 +7,14 @@
 from player import CharacterSprite
```

Comment:
Great job here both reorganizing and adding some good documentation. :+1: :high_heel: